js/web/lib/onnxjs/util.ts

```diff
@@ -426,6 +426,8 @@ export class ProtoUtil {
         return onnx.TensorProto.DataType.INT32;
       case 'uint32':
         return onnx.TensorProto.DataType.UINT32;
+      case 'float16':
+        return onnx.TensorProto.DataType.FLOAT16;
```

Comment:
WebGL does not support float16 by just adding this line. It's a better idea to revert this line in this PR

Reply:
(related) Hopefully it will soon :). "This proposal is at Stage 3 of [The TC39 Process](https://tc39.es/process-document/) as of the May 2023 meeting. It awaits implementations." https://github.com/tc39/proposal-float16array